### DISCUSSION TITLE
Support for AIOSEO Plugin

### DISCUSSION THREAD

**Discussion by CodingSamurai** [publishpress/publishpress-checklists#525] [Feature Requests]
Require a minimum SEO Analysis score to publish: https://aioseo.com/features/seo-audit-checklist/

Require a minimum Headline Analyzer score to publish: https://aioseo.com/docs/using-the-headline-analyzer-in-all-in-one-seo/

**Comment by stevejburge** [comment]
Solved in 2.18
